build.yaml

```diff
@@ -221,7 +224,8 @@ common_cli:
     export FLOCKER_FUNCTIONAL_TEST_OPENSTACK_REGION=dfw
 
   setup_coverage: &setup_coverage |
-    parse_logs pip install coverage==3.7.1 http://data.hybridcluster.net/python/coverage_reporter-0.01_hl0-py27-none-any.whl  --trusted-host \$TRUSTED_HOST
+    parse_logs pip install coverage  \
+      -i \$PIP_INDEX_URL --trusted-host \$TRUSTED_HOST
 
```

Comment:
installing coverage the normal way, there was probably a reason for coverage to be pulled from data.hybridcluster.net, but tests run while pulling it directly from pip